src/mol-repr/structure/visual/util/link.ts

```diff
@@ -80,7 +80,7 @@ export function calculateShiftDir(out: Vec3, v1: Vec3, v2: Vec3, v3: Vec3 | null
 
 export interface LinkBuilderProps {
     linkCount: number
-    position: (posA: Vec3, posB: Vec3, edgeIndex: number) => void
+    position: (posA: Vec3, posB: Vec3, edgeIndex: number, unadjusted?: boolean) => void
```

Comment:
I think having this as optional param could cause compiler de-optimization for this function since the signature could change between calls. Could you make this a mandatory unadjusted: boolean and be explicit with the true/false in the function calls?

Reply:
I thought this would impact a lot of code, but it doesn't because most existing builder props can just ignore this extra argument. I'll push an update changing this into a positive verb `adjust`.